simplq/src/components/common/QueueHistory/HistoryRow.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import './QHistory.css';
+import moment from 'moment';
```

Comment:
Can you use scss files for styles and use them the same was as in other scripts? We want to keep things consistent. Also please rename QHistory to QueseHistory.module.scss.

So use import styles from './QueueHistory.module.scss';

Reply:
Sure, I have resolved it in my latest commit e301f52